images/dev.Dockerfile

```diff
@@ -25,14 +25,53 @@ baseurl=https://repo.goreleaser.com/yum/ \n\
 enabled=1 \n\
 gpgcheck=0' > /etc/yum.repos.d/goreleaser.repo
 
-# Install proper version of nodejs so that make web-build works
-RUN dnf module install -y nodejs:20
-
 # Install goreleaser and various other packages we need
-RUN yum install -y goreleaser npm xrootd-devel xrootd-server-devel xrootd-client-devel nano xrootd-scitokens \
```

Comment:
For the "dev" container, I'd like to keep an entire, valid development environment.

So, let's not do the split here like we do in the production container.

This means that the development container will indeed grow significantly -- but I'm OK with that.